css/templates/travel/page-vars.css

```diff
@@ -0,0 +1,229 @@
+/**
```

Comment:
why do we have a CSS file inside templates? 
I believe there is already a page-vars checked in for travel, can you check on the project structure and make sure it works

Reply:
It's in the css directory, (`css/templates`).

Reply:
Why does it not show the diff then? there is already a file existing the the same place? Can you sync please

Reply:
Sure. I'll sync and open up another PR.

Reply:
I think just sync and push this PR should be fine. (dont have to do another PR)